### ISSUE TITLE
[Request deployment] New Hub: binder.hhmi.2i2c.cloud

### ISSUE THREAD

**Issue by jmunroe** [2i2c-org/infrastructure#3655]
A part of https://github.com/2i2c-org/infrastructure/issues/4109.

## Phase 2: Cluster

Use the existing hhmi cluster.

#### Phase 3.1: Initial setup (**READY**)

| Question | Answer | Notes |
| - | - | - |
| Name of the hub | binder | |
| Dask gateway? | no | |
| URL | binder.hhmi.2i2c.cloud |  | 
| BinderHub UI? | yes | | 
| Authentication Mechanism | None | |
| Admin Users | NA | Because BinderHub UI will be used | 
| Persistent Home Directory | no ||

#### Phase 3.2: Object storage access (**READY**)

| Question | Answer | Notes |
| - | - | - |
| Scratch bucket enabled? |  no |  |
| Persistent bucket enabled? | no |  |
| Requestor pays requests to external buckets allowed? | no | |  

#### Phase 3.3: Profile List (**READY**)

N/A, as BinderHub UI will be used

#### Phase 3.31: Resource configuration (**READY**)

No profile list, properties set under `singleuser`

| Question | Answer |
| - | -|
| memory limit | 256M |
| memory request | 1G |
| CPU limit | 2 |
 
#### Phase 3.4: Authentication tuning (**READY**)

| Question | Answer |
| - | - |
| Authentication Mechanism | None |

#### Phase 3.5: Profile List finetuning (**NA**)

NA, as BinderHub UI will be used

#### Phase 3.6: BinderHub config (**NOT READY**)

| Question | Answer |
| - | - |
| Allowed Repositories? | `^binder-examples/.*|LorenFrankLab/.*$` |
| Banner Message | "" | 
| BinderHub About Message | "" |

**Comment by jmunroe** [comment]
I appreciate the @consideRatio breakdown in https://github.com/2i2c-org/meta/issues/804 of functionality. To make this request more specific, I think this new `binder.hhmi.2i2c.cloud` should enable

- Image building -- YES
- Anonymous users -- YES
- Authenticated users -- NO
- User storage -- NO
- dask-gateway -- NO (@colliand ? I am not sure about this one)
- magic-links -- YES

The existing `hhmi.2i2c.cloud` should have 

- Image building -- YES
- Anonymous users -- NO
- Authenticated users -- YES
- User storage -- YES
- dask-gateway -- YES (@colliand ? I am not sure about this one)
- magic-links -- YES

And for completeness, the hub `spyglass.hhmi.2i2c.cloud` recently deployed has

- Image building -- NO
- Anonymous users -- YES
- Authenticated users -- NO
- User storage -- NO
- dask-gateway -- NO
- magic-links -- YES
- MySQLdb sidecar container -- YES

Does specifying hubs with this list of requested functionality help give flexibility to @2i2c-org/engineering ?

**Comment by haroldcampbell** [comment]
This can be delay in favour of the catalyst hubs. @jmunroe shared that perspective on [slack](https://2i2c.slack.com/archives/C055A1J1DRP/p1709313097763919?thread_ts=1709237695.196689&cid=C055A1J1DRP).

**Comment by yuvipanda** [comment]
I spoke to @jmunroe and it's time to pick this up again.

I have reformatted this to use the newer format for hub requests.

**Comment by sgibson91** [comment]
I tried to deploy this in https://github.com/2i2c-org/infrastructure/pull/4320 but authentication to the image registry is failing. I tried GAR instead of quay.io because it is a GCP cluster. I also tried implementing https://github.com/2i2c-org/binderhub-service/issues/113 incase that is specific to GAR binders (the binderhub-ui-demo hub is also on GCP), but that still didn't work.

We should timebox this and if not reasonable solution can be found, I'll just use quay.io instead.
